src/scripts/main.js

```diff
@@ -1,3 +1,44 @@
 'use strict';
 
 // write code here
+
+const tree = document.querySelector('.tree');
+
+treeRecursion([tree][0]);
+
+
+
+function treeRecursion(elem) {
+
+  if (elem.children.length !== 0) {
```

Comment:
just to decrease nesting a bit

```suggestion
  if (!elem.children.length) {
     return
  }
  
  .....
```